buildSrc/src/main/java/org/elasticsearch/gradle/tool/DockerUtils.java

```diff
@@ -208,14 +208,24 @@ private static Result runCommand(Project project, String... args) {
         ByteArrayOutputStream stdout = new ByteArrayOutputStream();
         ByteArrayOutputStream stderr = new ByteArrayOutputStream();
 
-        final ExecResult execResult = project.exec(spec -> {
-            // The redundant cast is to silence a compiler warning.
-            spec.setCommandLine((Object[]) args);
-            spec.setStandardOutput(stdout);
-            spec.setErrorOutput(stderr);
-        });
-
-        return new Result(execResult.getExitValue(), stdout.toString(), stderr.toString());
+        try {
+            final ExecResult execResult = project.exec(spec -> {
+                // The redundant cast is to silence a compiler warning.
+                spec.setCommandLine((Object[]) args);
+                spec.setStandardOutput(stdout);
+                spec.setErrorOutput(stderr);
+            });
+            return new Result(execResult.getExitValue(), stdout.toString(), stderr.toString());
+        } catch (GradleException ex) {
+            final String message = String.format(
+                Locale.ROOT,
+                "Error occurred running command [%s] stderr: [%s] stdout: [%s]",
```

Comment:
I think it would be more readable with stderr and stdout moved to their own lines.